lib/pluginmanager/package.rb

```diff
@@ -0,0 +1,32 @@
+class LogStash::PluginManager::Package < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => false
+  option "--zip", :flag, "compress package as a zip file", :default => false
+  option "--clean", :flag, "clean up the generated dump of plugins", :default => false
+
+  def execute
+    signal_error("File #{LogStash::Environment::GEMFILE_PATH} does not exist or is not writable, aborting") unless File.writable?(LogStash::Environment::GEMFILE_PATH)
+    puts("packaging plugins for offline usage")
+    LogStash::Bundler.invoke!({:package => true, :all => true})
+    compress_cache(LogStash::Environment::CACHE_PATH)
+    rm_rf(LogStash::Environment::CACHE_PATH) if clean?
```

Comment:
I am not sure I understand this. why would you want to generate the cache and clean it right after?